students/luftsmeerflier/lesson05/mailroom_3.py

```diff
@@ -96,15 +99,24 @@ def exit_program():
 	"2 - Create a Report",
 	"3 - Send letters to all donors.",
 	"4 - quit"
-))
+	))
 
 
 def main():
 	while True:
-		response = int(input(prompt + "\n")) # continuously colelct user selection
-		# now redirect to feature functions based on the user selection
-		arg_dict.get(response)()
+		response = int(input(prompt + "\n")) # continuously collect user selection
+		try:
+			isinstance(response, int)
+			reponse > 0
+			response < 5
```

Comment:
this statements seem to have no effect? also, `reponse` is misspelled and crashes the program.